metrics/chronos_httpserver.nim

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2019-2021 Status Research & Development GmbH
+# Licensed and distributed under either of
+#   * MIT license: http://opensource.org/licenses/MIT
+#   * Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+# at your option. This file may not be copied, modified, or distributed except according to those terms.
+
+################################
+# HTTP server (for Prometheus) #
+################################
+
+when defined(nimHasUsed):
+  {.used.}
+
+import net
```

Comment:
For what reason `net` has been imported without `when defined(metrics)` ?
There a lot of trash inside `net.nim`.
Also this module is from stdlib so it should be imported with `std/` prefix.

Reply:
The `net` module in needed for the `Port` argument of `startMetricsHttpServer()`.